ports/stm32/boards/STM32H747I_DISCO/mpconfigboard.h

```diff
@@ -0,0 +1,304 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2025
+ */
+
+#define MICROPY_HW_BOARD_NAME       "STM32H747I-DISCO"
+#define MICROPY_HW_MCU_NAME         "STM32H747"
+#define MICROPY_HW_FLASH_FS_LABEL   "STM32H747I-DISCO"
+
+// Network config
+#define MICROPY_PY_NETWORK_HOSTNAME_DEFAULT "mpy-stm32h747i-disco"
+
+#define MICROPY_OBJ_REPR            (MICROPY_OBJ_REPR_C)
+typedef int mp_int_t;               // must be pointer size
+typedef unsigned int mp_uint_t;     // must be pointer size
+
+#define MICROPY_FATFS_EXFAT         (1)
+#define MICROPY_HW_ENABLE_RTC       (1)
+#define MICROPY_HW_ENABLE_RNG       (1)
+#define MICROPY_HW_ENABLE_ADC       (1)
+#define MICROPY_HW_ENABLE_DAC       (1)
+#define MICROPY_HW_ENABLE_USB       (1)
+#define MICROPY_HW_HAS_SWITCH       (1)
+#define MICROPY_HW_HAS_FLASH        (1)
+#define MICROPY_HW_ENABLE_SERVO     (1)
+#define MICROPY_HW_ENABLE_TIMER     (1)
+#define MICROPY_HW_ENABLE_SDCARD    (1)
+#define MICROPY_HW_ENABLE_MMCARD    (0)
+#define MICROPY_HW_ENTER_BOOTLOADER_VIA_RESET   (0)
```

Comment:
Did you find this option was necessary?  It's best to go via reset if possible, to reset WDT.